### ISSUE TITLE
[BUG] Navigation Menu - Expanded state must be programmatically determined by screen reader

### ISSUE THREAD

**Issue by alfredoscaini** [wet-boew/wet-boew#9557]
We have an accessibility resource reviewing our front end websites and she noted that when a user expands the menu using the ‘space’ key it must announce the expanded state using aria-expanded=”true”

    Environment
    *Url:* 
      https://www.canada.ca/en.html

*Browser:*
  Firefox version 91.8.0esr (64-bit)
  Chrome version 100.0.4896.127 (Official build) (64-bit)

*Tool used:*  
  Keyboard navigation and NVDA screen reader

    To Reproduce

1. Go to:  https://www.canada.ca/en.html
2. Using ‘tab’ (keyboard navigation) set focus on the navigation menu.
3. Click the ‘space’ key to expand.
4. Menu does not announce its expanded state.

    Expected Behaviour
    Must announce the expanded state using aria-expanded=”true”

    Noted Results
    Does not announce the expanded state using aria-expanded=”true”

    Remediation: 
     Ensure when the menu’s state changes the user is notified.  

    Reference 
    *WCAG 2.0*
    Success Criterion 4.1.2 Name, Role, Value (Level A)
